main/guides/js-programming/far.md

```diff
@@ -0,0 +1,95 @@
+
+# `Far()` and Remotable Objects
+
+## Remotable and passable objects
+
+In Agoric smart contracts and dapps, you can call methods on objects from other vats or machines. 
+For example, a purse for an ERTP issuer actually lives in the issuer's vat. But code in your off-chain
+wallet or another contract can still use that purse.
+
+This is possible because Agoric encapsulates inter-machine and inter-vat communication. At
+the smart contract or dapp level, objects from other vats or machines can almost be treated 
+as if they are local.
+
+To call a method on an object from another vat or machine, you must 
+use [`E()`](./eventual-send.html#remote-object-communication-with-e). For example, getting
+an `Issuer`'s `brand` would look like `E(issuer).getBrand()`.
+
+Objects intended to be used in other vats are called *remotables*. Remote messages sent to
+remotables must only contain *passable* arguments and return *passable* results. 
+Passables includes all things that can be passed as arguments in messages. 
+
+There are three kinds of passables:
+   * Remotables, objects with methods that can be called remotely using `E()`.
+   * Pass-by-copy data, such as numbers or hardened records
+   * Promises for passables.
+
+In particular, note that every object returned from a smart contract, such a `publicFacet` or 
+`creatorFacet`, must be passable. All objects used in your contract's external API must
+be passable.
+
+### Rules for creating remotables
+- All property values must be functions. 
+  - They cannot be accessors.
+  - Note: If you wish to send data, send pass-by-copy data such as pass-by-copy records, 
+    pass-by-copy arrays, strings, numbers, etc. or some other passable.
```

Comment:
I think this bullet is more confusing than clarifying. The point it tries to make is already better made above. Please delete it.
```suggestion
```